### PR TITLE
Cancel

### PR DESCRIPTION
**Pull request by rgrunber**

### CODE REVIEW

**Comment by mickaelistria** [comment]
Please also bump the minor version when moving to API incompatible LemMinX. Also, if LemMinX breaks API, it'd be better to bump the minor segment, not just the service/patch segment.

File: lemminx-maven/pom.xml [hunk 1]
```diff
@@ -22,7 +22,7 @@
 		<lemminx.maven.indexDirectory>${project.build.directory}/test-index</lemminx.maven.indexDirectory>
 		<maven.repo.local>${settings.localRepository}</maven.repo.local>
 		<junit-jupiter.version>5.7.0</junit-jupiter.version>
-		<lemminx.version>0.15.0</lemminx.version>
+		<lemminx.version>0.15.1-SNAPSHOT</lemminx.version>
```
Comment (datho7561) [inline lemminx-maven/pom.xml:25]: Probably worthwhile to upgrade to LemMinX 0.16.2 now

**Review by datho7561 — CHANGES_REQUESTED** [review]
I had to change the source and target level to 8 to get it to work with LemMinX. Some of the tests are failing.

File: lemminx-maven/pom.xml (hunk 1 above)
Comment (rgrunber) [reply lemminx-maven/pom.xml]: Updated to 0.16.2

**Comment by mickaelistria** [comment]
That looks all good to me, and to CI. Let's merge.
Thanks!

**Pull request merged**